common-lib/src/cmr/common/test/runners/ltest.clj

```diff
@@ -0,0 +1,38 @@
+(ns cmr.common.test.runners.ltest
+  "This test runner is a CMR adaptor for the ltest Clojure test runner."
+  (:require
+   [clojure.stacktrace :as stack]
+   [clojure.string :as string]
+   [clojure.test :as test]
+   [cmr.common.test.runners.util :as util]
+   [ltest.core :as ltest]
+   [potemkin :refer [import-vars]]))
+
+(import-vars
```

Comment:
Yeah, in general, potemkin lets one maintain a user-facing API without being tied to a particular namespace structure and layout (by importing vars).

In our case:

* we get a new home with the real code that gets updates/maintenance/etc., while we can forget the deprecated ns (the `test-runner` namespace)
* for this namespace, we get to pull in the var itself (including docstring, so this would show up in codox) which means that users can call `ltest/run-test` and `ltest/run-tests` without us having to implement a wrapper in the namespace (which, really, ties in with the API maintenance point I made above)